package.json

```diff
@@ -27,6 +27,7 @@
     "build:docs": "node ./scripts/generateDocs.js",
     "build": "npm run tsify:xterm-css && npm run build:cjs && npm run build:esm5 && npm run build:docs",
     "generate-screenshots": "./scripts/generate-screenshots.sh",
+    "update-snapshots": "npm run build && npm run jest -- -u",
```

Comment:
it would be nice if we had the same script prefix for screenshot & snapshot update/generation.

Reply:
Like this?
```suggestion
    "generate-snapshots": "npm run build && npm run jest -- -u",
```

Reply:
I was trying to be closer to the real action that is happening, but I agree, it is better to be consistent.